SimG4Core/Generators/src/Generator.cc

```diff
@@ -130,13 +133,9 @@ void Generator::HepMC2G4(const HepMC::GenEvent * evt_orig, G4Event * g4evt)
                                      (*(evt->vertices_begin()))->position().z(),
                                      (*(evt->vertices_begin()))->position().t());
 
-  if(verbose > 0) {
-    edm::LogVerbatim("SimG4CoreGenerator") << &evt;
```

Comment:
@civanch this might help in understanding the generator structure to be imported, was it disturbing?

Reply:
@fabiocos , when we debug various problems it become useful to enable edm::LogVerbatim to see list of primary vertexes/particles. Here I tried make the printout to be easier to find in a long log and more convenient for reading. The old style "verbose" variable requires change of g4SimHits parameter (but save a drop of CPU). With this change debugging is a bit easier, however, this is a minor thing.